core/src/test/scala/org/broadinstitute/dsde/rawls/dataaccess/slick/DataAccessSpec.scala

```diff
@@ -70,4 +70,21 @@ class DataAccessSpec extends TestDriverComponentWithFlatSpecAndMatchers with Sca
       }
     }
   }
+
+  it should "use utf8 for MySQL's character_set_server" in withEmptyTestDatabase {
+    /* Our live-environment CloudSQL instances, including production, use character_set_server=utf8.
+       This setting is critical for SQL queries that specify/override a collation, such as to make a
+       case-sensitive query against a column that is case-insensitive by default, e.g. the column
+       uses utf8_general_ci collation.
+
+       A failure of this test likely means that our test environment is not set up correctly; the mysql
+       instance used by tests does not have the right setting. If the test-instance mysql is not set up
+       correctly, other tests can fail and/or return false positives.
+     */
+    val charsetLookup = runAndWait(sql"""SHOW VARIABLES WHERE Variable_name = 'character_set_server';""".as[(String, String)])
+    charsetLookup should have size 1
+    withClue("is the mysql against which these unit tests ran set up correctly with --character-set-server=utf8 or equivalent?") {
+      charsetLookup.head._2 shouldBe "utf8"
+    }
+  }
```

Comment:
I verified that this test fails ("latin1 was not equal to utf8") without the changes to `docker/run-mysql.sh`